combustion-android-ble/src/main/java/inc/combustion/framework/ble/NetworkManager.kt

```diff
@@ -398,16 +397,49 @@ internal class NetworkManager(
     }
 
     private suspend fun collectAdvertisingData() {
-        DeviceScanner.advertisements.collect {
+        DeviceScanner.advertisements.collect {advertisingData ->
             if(scanningForProbes) {
-                if(it.probeSerialNumber != REPEATER_NO_PROBES_SERIAL_NUMBER) {
-                    if(manageMeatNetDevice(it)) {
+                if(advertisingData.probeSerialNumber != REPEATER_NO_PROBES_SERIAL_NUMBER) {
+                    if(manageMeatNetDevice(advertisingData)) {
                         mutableDiscoveredProbesFlow.emit(
-                            ProbeDiscoveredEvent.ProbeDiscovered(it.probeSerialNumber)
+                            ProbeDiscoveredEvent.ProbeDiscovered(advertisingData.probeSerialNumber)
                         )
                     }
                 }
+                else {
+                    if(firmwareStateOfNetwork[advertisingData.id] == null) {
+                        handleTimerWithoutProbes(advertisingData)
+                    }
+                }
+            }
+        }
+    }
+
+    private fun handleTimerWithoutProbes(advertisement: CombustionAdvertisingData) {
```

Comment:
This probably ought to be `handle[Repeater|MeatNetNode|etc]WithoutProbes` or whatever terminology we're using for that class of device. I can pick that up in a later PR if you'd like.